server/form.php

```diff
@@ -5,20 +5,30 @@
 	$retrieve = $_GET['retrieve'];
 
 	if ($retrieve == "questions") {
-		// TODO make this handle multiple locations, if necessary
-		$questionStatement = $conn->prepare('SELECT questionId, string, inputType, placeholder, tag, subheading, required, archived FROM Question
-			WHERE (archived != true)
+		$subheadings = [];
```

Comment:
This block of subheadings code seems awfully convoluted for what amounts to filling an array. Could we not try to guarantee that `$_GET['subheadings']` will always return an array.

More conditionals means slower page load. While this isn't a significant time suck, if it's easy to fix, fix it.